TLSharp.Tests.VS/TLSharpTestsVs.cs

```diff
@@ -0,0 +1,71 @@
+﻿
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TLSharp.Tests
+{
+    [TestClass]
+    public class TLSharpTestsVS : TLSharpTests
```

Comment:
Best Practice is to create SharedLibrary for Tests and call them in NUnit or MSTest